assignments/function-conversion.js

```diff
@@ -1,24 +1,18 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+const myFunction = () => {
+console.log("Function was invoked!");
+};
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+const anotherFunction = param => console.log(`${param}`);
+anotherFunction("Example");
 
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
+const add = (param1, param2) => console.log(`${param1 + param2}`);
```

Comment:
Remember there's a difference between `console.log` and `return`. `console.log` allows you to print some output to some external display, the console. You can see that console in the browser console or in a terminal in node for example. `console.log` does _not_ determine what the output of a function is.

In contrast, `return` signals the end of the function's execution, and any expression after it is designated to be the output of the function.

Why is this important? Well, looking at your function, you `console.log` the output instead of returning. This is OK in isolation:

```js
const add = (param1, param2) => console.log(`${param1 + param2}`);
add(1,2); // 3 is logged to the console
```

But what about if I need to use the output of add?

```js
const add = (param1, param2) => console.log(`${param1 + param2}`);
console.log(add(1,2) / add(3,3)); // this outputs NaN (not a number)
// 3 and 6 is output to the console
```

Because the function doesn't return anything, I'm unable to use the output of add for other operations. That severely limits the usefulness of the function. Instead, have the functions return their value, so you can use those values!

```js
const add = (param1, param2) => param1 + param2;
console.log(add(1,2));            // console.logs 3
console.log(add(1,2) / add(3,3)); // console.logs 0.5
```

Unless the exercise specifically tells you to, assume the output is to be returned, and `console.log` it afterwards.

Reply:
ah got it, this was a quick work around to get an output that I can see in the console. But I see your point, will edit this